recipe/meta.yaml

```
@@ -23,7 +23,7 @@ requirements:
     - pip
   run:
     - python >=3.7
-    - conda-build >=3.24
+    - conda-build >=3.25
```

We should add
```suggestion
    - conda-build >=3.25,<24.5
```
or at least
```suggestion
    - conda-build >=3.25,<24.7
```
. IMO, it is better (conceptually and technically) to relax rather than restrict later.

(N.B.: Generally, I have a rather stern opinion on this, in that I believe projects like `boa`/`mamba 1.*` should always tightly bind their `conda`/`conda-build` deps. By using non-public-API functionality, one buys into/has to accept the necessary maintenance work of closely following upstream changes.)